docs/docs/getting-started/bare-metal.md

```diff
@@ -39,21 +39,40 @@ Increase the `user.max_inotify_instances` sysctl limit by adding `user.max_inoti
 <TabItem value="amd" label="AMD SEV-SNP">
 To enable GPU usage on a Contrast cluster, some conditions need to be fulfilled for *each cluster node* that should host GPU workloads:
 
-1. You must activate the IOMMU. You can check by running:
+1. Ensure that CC-capable GPUs are available on the machine.
+
+   ```sh
+   lspci -nnk | grep '3D controller' -A3
+   ``
+
+   which should show a [CC-capable](https://www.nvidia.com/en-us/data-center/solutions/confidential-computing/) GPU like the NVIDIA H100:
+
+   ```console-out
+   41:00.0 3D controller [0302]: NVIDIA Corporation GH100 [H100 PCIe] [10de:2331] (rev a1)
+      Subsystem: NVIDIA Corporation GH100 [H100 PCIe] [10de:1626]
+      Kernel driver in use: vfio-pci
+      Kernel modules: nvidiafb, nouveau
+   ```
+
+   :::warning
```

Comment:
I'd reserve warning for the severity of security problems or similar. `note` or `info` sound important enough.